### PR TITLE
Add meeting id & password at email approval reservation

### PR DESCRIPTION
**Pull request by iprasetyaa**
# Overview

Add meeting id & password at email approval reservation recurring and single

CC :  @jabardigitalservice/jds-backend

### CODE REVIEW

File: app/Listeners/CreateZoomMeetingRecurring.php [hunk 1]
```diff
@@ -41,20 +41,20 @@ public function handle(AfterReservationRecurringCreated $event)
         foreach ($firstReservation as $item) {
             $reservation = Reservation::where('id', $item['id'])->first();
             $asset = $reservation->asset;
-            $userZoom = null;
```
Comment (samudra-ajri) [inline app/Listeners/CreateZoomMeetingRecurring.php:44]: Punten, Kang..
Kira2 klw variablenya jadi `$zoomUser` gimana, Kang? Karena penulisan variable yg lain zoom-nya disimpen di depan (`$zoomDay $zoomData`). :grin:

File: app/Listeners/CreateZoomMeetingRecurring.php [hunk 2]
```diff
@@ -41,20 +41,20 @@ public function handle(AfterReservationRecurringCreated $event)
...
+            $zoomData = null;
```
Comment (iprasetyaa) [inline app/Listeners/CreateZoomMeetingRecurring.php:44]: @samudra-ajri aku udah rubah jadi zoomData

Comment (samudra-ajri) [reply app/Listeners/CreateZoomMeetingRecurring.php:44]: wkwkw... maaf, Kang..
Lg konslet golang kayanya, nih.. hhe

**Review by samudra-ajri — APPROVED** [review]

File: app/Listeners/CreateZoomMeetingRecurring.php (hunk 2 above)
Comment (iprasetyaa) [reply app/Listeners/CreateZoomMeetingRecurring.php:44]: mantap euy @samudra-ajri  😆

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**